joomla/joomla-cms
# [3.10] Add back com_joomlaupdate default.min.js

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/31200#issuecomment-808710736 .

### Summary of Changes

Add back the minified js which has been removed with #31200 .

I've used https://jscompress.com/ to minify the js.

### Testing Instructions

Make sure that the Joomla Update Component javascript stuff (pre-update checker, quickicon) still works with the minified js.

### Actual result BEFORE applying this Pull Request

No minified js present.

### Expected result AFTER applying this Pull Request

Minified js is present and used, the Joomla Update component works as well as before.

### Documentation Changes Required

None.
